tests/test_utils.py

```diff
@@ -101,3 +102,32 @@ def test_flatten_dict(self):
         d = dict(A=1, B=2, C=dict(A=3, B=4, C=dict(A=5, B=6)))
         e = dict(A=1, B=2, C_A=3, C_B=4, C_C_A=5, C_C_B=6)
         self.assertEquals(flatten_dict(d, sep='_'), e)
+
+    def test_parse_env_tags(self):
```

Comment:
pytest parametrized tests are great for this kind of stuff.

as an example: https://github.com/DataDog/dd-trace-py/blob/a6653c383d23a8e5740a8c79411d3fe352fe8467/tests/test_span.py#L444-L465

Reply:
(not a requirement, just a thought)